add_options.rb

```diff
@@ -41,4 +44,33 @@ def add_music_album
     print ENTR_MSG
     gets
   end
+
+  def add_game
+    puts 'Pleae enter the publish date of the game:'
```

Comment:
Please be careful not to print typos into the UI. ☝️

Reply:
You are right. I haven't focused on the message. I just focused more on the performance of the method. I will work on it.